test/e2e/kubectl.go

```diff
@@ -0,0 +1,215 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
```

Comment:
Whoops, fixed. I now wonder other files were recently added with old copy of the boilerplate.